File: springbootjparest/src/main/java/com/scarc/springbootjparest/models/Company.java
package com.scarc.springbootjparest.models;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.List;

@Builder
@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIdentityInfo(
    generator = ObjectIdGenerators.PropertyGenerator.class,
    property = "id")
public class Company {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private String name;
    private String zipTown;
    private String street;
    private String phone;
    private String eMail;
    private String replyTo;
    private String comments;

    @OneToMany(mappedBy = "company")
    private List<Participation> participatesIn;
}

File: KOTLIN/Gstohlen/Kotlin_Wirtschaftstag/app/src/main/java/at/eder/wirtschaftstagmobileapp/models/Company.kt
package at.scarc.androidApp.models

data class Company(
    var id: Long?,
    var name: String?,
    var zipTown: String?,
    var street: String?,
    var phone: String?,
    var email: String?,
    var replyTo: String?,
    var comments: String?
) {
    override fun toString(): String = "$id - $name: $zipTown"
}